kubernetes/cray-sysmgmt-health/Chart.yaml

```diff
@@ -6,4 +6,4 @@ appVersion: 9.3.1
 description: An extension of the official prometheus-operator helm chart for monitoring system health.
 name: cray-sysmgmt-health
 home: "cloud/cray-charts"
-version: 0.17.0
+version: 0.17.1
```

Comment:
This adds a new alert, so according to semver "MINOR version when you add functionality in a backwards compatible manner" this should be incrementing the MINOR version, see https://semver.org/

Reply:
changed version to 0.18.0